code/modules/mob/inventory.dm

```diff
@@ -212,3 +222,25 @@
 		if(slot_r_hand)
 			return r_hand
 	return null
+
+// Called when a mob moves from a spot where dropping is forbidden, to a spot
+// where dropping is allowed - so taking off your jumpsuit in a mech doesn't let
+// your ID stay on - or let you carry a large O2 tank in suit storage without
+// the hardsuit itself
+/mob/proc/update_item_drops()

```

Comment:
Just to be clear, this means that if someone jumps out of a mech, all slots are checked to see if they are inside of an invalid location?

So you take off your jumpsuit in a mech, and once you jump out it drops everything on the ground? How's that currently handled?